common/src/main/java/com/netflix/conductor/common/metadata/tasks/Task.java

```diff
@@ -206,6 +206,12 @@ public boolean isRetriable() {
     @ProtoField(id = 43)
     private String taskDescription;
 
+    @ProtoField(id = 44)
+    private int publishCount;
```

Comment:
Task is one of the base entity, this might create merge conflict while taking changes from upstream Netflix branch. Is it possible to do it some other way?

Reply:
We need to create a state for last published time. It will be too much to create new object and create ways to store them in redis for these two properties.

Reply:
One option is to introduce custom meta attributes map in Task entity. And that change can be pushed to main line Netflix/Conductor.

Reply:
I am not sure whether the community will accept it